### PR TITLE
Enable STUN by default

### PR DESCRIPTION
**Pull request by Stell0**

### CODE REVIEW

**Review by gsanchietti — CHANGES_REQUESTED** [review]
Property names should be in camel case, please change the name of following props:
- natMode -> NatMode
- publicIP -> PublicIp 
- stunPort -> StunPort
- stunServer -> StunServer

File: root/etc/e-smith/templates/opt/janus/etc/janus/janus.cfg/10base [hunk 1]
```diff
@@ -90,7 +98,15 @@ nice_debug = false
 ; such, use the option with caution and only if you know what you're doing.
 ; Besides, it's still recommended to also enable STUN in those cases,
 ; and keep ICE Lite disabled as it's not strictly speaking a public server.
-;nat_1_1_mapping = 1.2.3.4
+
+{
+    my $mode = ${'janus-gateway'}{natMode} || 'stun';
+    if ($mode eq '1:1') {
+        my $ip = ${'janus-gateway'}{publicIP} || die ('publicIP missing');
```
Comment (gsanchietti) [inline root/etc/e-smith/templates/opt/janus/etc/janus/janus.cfg/10base]: Try to not fail hard here.

I'd change the line 104 with something like:
```
if ($mode eq '1:1' && $publicIp ne '') {
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]

**Review by gsanchietti — APPROVED** [review]

**Comment by Stell0** [comment]
nethserver-testing nethserver-janus-1.0.0-1.2.g158d2d6.ns7.noarch.rpm

**Comment by Stell0** [comment]
NethServer/dev#5373

**Pull request merged**